urdf_model_state/include/urdf_model_state/model_state.h

```diff
@@ -49,6 +49,15 @@
 
 namespace urdf{
 
+//round is not defined in C++98
+//So in Visual Studio is necessary to define it
+#ifdef _MSC_VER
```

Comment:
I think that we should add a check of version on this. I guess that `round` was eventually implemented in some recent versions of Visual Studio for C++11 compatibility ( http://www.cplusplus.com/reference/cmath/round/ ).